wp-admin/js/widgets/media-widgets.js

```diff
@@ -200,7 +204,7 @@ wp.mediaWidgets = ( function( $ ) {
 				 *
 				 * @return {void}
 				 */
-				createStates: function() {
+				createStates: function createStates() {
```

Comment:
Yes. The reason for naming the functions is for stack traces. Compare:

![image](https://cloud.githubusercontent.com/assets/134745/24719317/234015f0-19ee-11e7-97b5-0525719c660f.png)

More reasons here: https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch3.md#anonymous-vs-named